post_installation_configuration/coreos-layering.adoc

```diff
@@ -1,15 +1,20 @@
 :_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
 [id="coreos-layering"]
 = {op-system} image layering
-include::_attributes/common-attributes.adoc[]
 :context: coreos-layering
 
 toc::[]
 
 
 {op-system-first} image layering allows you to easily extend the functionality of your base {op-system} image by _layering_ additional images onto the base image. This layering does not modify the base {op-system} image. Instead, it creates a _custom layered image_ that includes all {op-system} functionality and adds additional functionality to specific nodes in the cluster.
 
-You create a custom layered image by using a Containerfile and applying it to nodes by using a `MachineConfig` object. The Machine Config Operator overrides the base {op-system} image, as specified by the `osImageURL` value in the associated machine config, and boots the new image. You can remove the custom layered image by deleting the machine config, The MCO reboots the nodes back to the base {op-system} image.
+[id="coreos-layering-about"]
```

Comment:
Per recent [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-subsections-level-1-and-lower-headings) update:
```suggestion
[id="coreos-layering-about_{context}"]
```